ramalama/cli.py

```diff
@@ -467,7 +468,23 @@ def run_container(args):
     exec_cmd(conman_args)
 
 
+def load_shortnames():
+    config = configparser.ConfigParser()
+    config_file = "/etc/ramalama/registries.conf.d/000-shortnames.conf"
```

Comment:
I should probably iterate through all the files in "/etc/ramalama/registries.conf.d/"